crml/cennzx-spot/src/lib.rs

```diff
@@ -729,7 +729,7 @@ impl<T: Trait> Module<T> {
 	}
 
 	/// `asset_id` - Trade asset
-	/// `amount_sold` - Amount of trade assets sold
+	/// `amount_sold` - Amount of the trade asset to sell
```

Comment:
@hoani @royyang-centrality @holygits Do you think I should revert this change? I now think I didn't get what that sentence meant (still not getting it to be honest)

Reply:
I think your comment is correct. Here we use Trade asset to buy Core, and we specify the amount of trade asset we are willing to use, and expect the function to return the amount of Core we can buy.
I.e. I have 100 sylo, tell me how much Cpay I can get out of this

Reply:
the qouted parameter should say `sell_amount`